docs/docs/reference/cloud/gatsby-cloud-environment-variables.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Environment Variables Specific to Gatsby Cloud"
+description: "Learn all about the special environment variables support to configure your Gatsby Cloud site"
+---
+
+The environment variables below allow you to configure features of Gatsby Cloud:
+
+- `NODE_VERSION`: Specify the version of Node.js your project should use. For example, `NODE_VERSION=10`. Defaults to `14`.
+- `NPM_TOKEN`: Use to access private npm modules.
+- `NPM_REGISTRY`: Use to set the URL of a private registry.
+- `YARN_FLAGS`: Flags that are passed through to the yarn command.
+- `NODE_OPTIONS`: Passed through as options for Node.js. For example, - `NODE_OPTIONS=--max-old-space-size=4096`.
+- `PREFIX_PATHS`: Set to true to enable the `--prefix-paths` flag during `gatsby build`. See the docs on [prefix-paths](https://support.gatsbyjs.com/hc/en-us/articles/360053096433).
```

Comment:
Shouldn't this link to our oss docs instead of zendesk?

Reply:
yes - good catch - updated.